### DISCUSSION TITLE
/introducing-vinyl-music-player/

### DISCUSSION THREAD

**Discussion by AdrienPoupa** [AdrienPoupa/adrien.poupa.net#5] [Comments]
We all listen to music on our smartphones. Since I&#8217;m old-fashioned, I still have a local library and no Spotify subscription. I&#8217;ve been using the Phonograph music player for a few years but after having see its development stopped for a few months, its creator decided to put some of its best features behind a [&hellip;]

https://adrien.poupa.net//introducing-vinyl-music-player/

**Comment by AdrienPoupa** [comment]
_From **Mykola** on **2018-01-22 15:10:24** (Migrated from Wordpress):_

Very neat music player. Thank you for the fork and greetings from Ukraine.

**Comment by AdrienPoupa** [comment]
_From **romain** on **2018-01-22 18:22:06** (Migrated from Wordpress):_

Hi what are the differences with vanilla music player ?

**Reply by AdrienPoupa** [reply]
_From **Adrien Poupa** on **2018-01-22 18:36:09** (Migrated from Wordpress):_

Hi,  
I am not familiar with Vanilla.  
They both seem to be relatively simple, lightweight and having a material design.

**Comment by AdrienPoupa** [comment]
_From **Ildar** on **2018-01-25 04:44:11** (Migrated from Wordpress):_

Adrien! 1st off Thanks A Lot for getting this player into F-Droid. I immediately switched to it for my daily music 🙂  
I’ve one question: what’s your plan about it? I see three options:  
1\. Liberation is done. Every new release would have patches applied and published. Relatively easy.  
2\. Critical and major issues would also be fixed.  
3\. Full development of the fork.  
?

**Reply by AdrienPoupa** [reply]
_From **Adrien Poupa** on **2018-01-25 22:24:55** (Migrated from Wordpress):_

Hello! Thanks for using it. For now, I plan to integrate upstream changes and eventually fix issues that would be reported, if I can. If there is a need for a new feature and if I’m able to implement it, why not.

**Comment by AdrienPoupa** [comment]
_From **Adrien Poupa** on **2018-01-28 20:06:26** (Migrated from Wordpress):_

I turned on Vinyl’s issue tracker so you can use it as well.

**Comment by AdrienPoupa** [comment]
_From **Dan** on **2018-01-28 14:19:53** (Migrated from Wordpress):_

Hey, I just stopped by to say thank you for this. I’ve been using VLC on Android for a long time because it’s a familiar app that had most of the things I needed from a music player, but after many problems with the latest version I went looking for another open source player and found Vinyl on F-Droid.

Man, I’m really impressed with it. I guess I never heard of Phonograph before because I keep my use of proprietary apps to an absolute minimum, but now that I tried Vinyl I must say this code is definitely something that should be maintained. Congrats on this initiative.

The only thing I’m hoping for is better album support. I listen to discographies chronologically, so a “sort by year” feature would be much appreciated. And artist view often shows an hyphen instead of the year for lots of albums that have their years correctly setup according to the tag editor.

Anyway, you’re awesome!

**Reply by AdrienPoupa** [reply]
_From **Adrien Poupa** on **2018-01-28 20:09:31** (Migrated from Wordpress):_

Hello,

Thanks for the comment! You can create an issue on the bug tracker and I’ll have a look.

**Comment by AdrienPoupa** [comment]
_From **SMike** on **2018-02-09 05:25:16** (Migrated from Wordpress):_

Just switched to Vinyl from Pulsar Music Player after they introduced ads, so far I love it!

**Comment by AdrienPoupa** [comment]
_From **Alex** on **2018-03-02 11:10:38** (Migrated from Wordpress):_

Hey Adrien

Thanks for your forked app. Is there any way we can compile this on our own PC?

**Comment by AdrienPoupa** [comment]
_From **khazar13** on **2018-03-22 18:03:43** (Migrated from Wordpress):_

Hi Adrien,

Superbe petit lecteur minimaliste, bien hébergé de plus 😉 Bô travail !

Question : Où est l’egaliseur (de son) ?

Quand je suis sur l’ecran de lecture du morceau et que je clique sous les 3 petits points verticaux, juste à droite du coeur, … j’ai un petit menu :  
………………………..  
– Effacer ….  
– Sauvevarder …  
– Minuteur …  
– Egaliseur  
………………………..  
et donc, quand je “clic” sur le mot égaliseur … “pouf” … rien! Juste un petit message qui me dit :  
“IMPOSSIBLE DE TROUVER D’EGALISEUR”  
Si tu sais me dire où le trouver?  
Merci!  
Best regards

khazar13

**Comment by AdrienPoupa** [comment]
_From **owo** on **2018-03-26 14:17:27** (Migrated from Wordpress):_

Hello!

Do you plan on adding Opus support ? I really like the app but since I mainly switched to Opus for my portable devices, only a part of my music collection gets played by the application.

Great app by the way. Slick design and oh so functional.

**Comment by AdrienPoupa** [comment]
_From **Jan** on **2018-04-13 19:57:16** (Migrated from Wordpress):_

after symbian died, looking for a whole time for this masterpiece!! THANX A LOT.!!! Love U 🙂 Love music &lt;3 Love LIFE!

**Comment by AdrienPoupa** [comment]
_From **daufinsyd** on **2018-04-29 10:15:18** (Migrated from Wordpress):_

Salut Adrien, je viens de découvrir ton appli en regardant les nouveautés F-Droid, elle est supebe ! (Elle a les qualités d’Apollo et de VLC + ses propres fonctionnalités et son design &lt;3)  
Enfin voilà, un grand merci à toi ! :+)

**Comment by AdrienPoupa** [comment]
_From **M Park** on **2018-04-29 15:03:10** (Migrated from Wordpress):_

Awesome software,  
Thanks so much.  
since ive lost my sony phone with sonys great music app, this player is the best one i have stumbled upon today through fdriod

**Comment by AdrienPoupa** [comment]
_From **da]v[ax** on **2018-05-04 00:58:28** (Migrated from Wordpress):_

Dude, you are my hero of the day. I loved phonograph a lot but this bitchmove with restricting the folder browser to the paid version really pissed me off. I kept an old version of phonograph running until I just discovered your neat little fork on F-Droid. THANK YOU SO MUCH!

**Comment by AdrienPoupa** [comment]
_From **da]v[ax** on **2018-05-04 07:03:25** (Migrated from Wordpress):_

You really made my day! I loved Phonograph a lot and would gladly have paid for it, but that bitch-move to strip the folder-view out of the free version really made me mad.

Congrats on your nice little fork! If I had a free wish: the widget-background should be dark instead of white, as all the widgets I use have a dark, semi-transparent background and Vinyl Music PLayer now sticks out like a sore thumb. I’ll file a request in Github for that.

Anyway: THANKS for Vinyl Music Player.

**Comment by AdrienPoupa** [comment]
_From **Jason** on **2018-07-05 13:51:26** (Migrated from Wordpress):_

I love the app – but I’ve noticed that the shuffle ends up picking the same songs. I have about 2000 songs on my device and it seems like I rarely get outside of about 150 of them. Any ideas?  
(great app)

**Comment by AdrienPoupa** [comment]
_From **Jesse** on **2018-07-18 20:39:44** (Migrated from Wordpress):_

What you’ve done is pretty much plagiarism. The paywall was to support development, so it could continue to be developed (what you were complaining about). You can’t fork over a measly amount to support another developer so you steal their code and make like it’s your own on the app store?

**Comment by AdrienPoupa** [comment]
_From **Kisuke** on **2018-07-20 05:07:04** (Migrated from Wordpress):_

Hello~!  
Vinyl Music Player is really great, it does the job and is great at it.  
I have two questions-suggestions about it if you don’t mind.  
– Can it play tracker files? (MOD, XM, S3M, IT and so on) If not, can it be added sometime in future?  
– Can something be done about MID files? I mean for example ability to put your own SF2 (SoundFont) file?

Thank you for advance and have a good day 🙂

Kisuke

**Comment by AdrienPoupa** [comment]
_From **andrew bennett** on **2018-07-31 18:26:47** (Migrated from Wordpress):_

hi, how do i sync with my google play music account.? using google pixelbook laptop.

**Comment by AdrienPoupa** [comment]
_From **fatalerrors** on **2018-08-03 09:04:47** (Migrated from Wordpress):_

Hi, I’m using it since a couple of days and I think it’s the best local music player so far… But on my version under lineageos the tag editor don’t write data after clicking the floppy

**Comment by AdrienPoupa** [comment]
_From **steph** on **2018-09-02 08:51:59** (Migrated from Wordpress):_

Salut Adrien,  
Merci pour cette belle application, smooth, efficace et légère.  
Est-ce qu’un plugin “lyrics” serait en gestation? 😉  
Bravo encore

**Comment by AdrienPoupa** [comment]
_From **Savatore** on **2018-10-18 14:17:11** (Migrated from Wordpress):_

Hi, I’m struggling to find an android player that sorts albums via the FULL year tag (YYYY-MM-DD).  
Is Vinyl capable to do so?  
Thanks 🙂

**Comment by AdrienPoupa** [comment]
_From **Jared** on **2018-11-21 01:24:06** (Migrated from Wordpress):_

This is the perfect music player! I use this app on a daily basis, and I can gladly say that, after trying countless other apps, this is definitely the best one. I’m surprised that more people haven’t downloaded it. Thanks so much for the fork!

**Comment by AdrienPoupa** [comment]
_From **Abraham Zamora M** on **2019-02-01 10:01:29** (Migrated from Wordpress):_

Hello, greetings from Aztec land. I really like Phonograph, when I tried it, I was amazed by its interface, very minimalist but, so beautiful in every sense of the word. When, later, I discovered Vinyl Music Player, I made this my favorite music player. The same of Phonograph but with the PRO options without restrictions. I really appreciate the work and improvements you have made to Vinyl, simply, I do not change it for anything. I would like to suggest a couple of things. First, see the possibility of resizing the RW, Beach / Pause and FF buttons; so that they are not so small and are more accessible to thick fingers. The second suggestion is to change and / or add the cover from the application, as does Stellio, T-Shirt PRO, Amp Power, etc. Finally, I would like to contribute with the translation of the app to the Spanish language, is there any way to contribute to it? Thank you and I send you cordial greetings and congratulations for such a wonderful job.

**Comment by AdrienPoupa** [comment]
_From **Tom** on **2019-02-27 01:47:24** (Migrated from Wordpress):_

Hey Adrian!  
In first place thank you a lot for this really nice piece of software.  
I have two questions:  
1\. since the last update in the artist view there are only pictures if available but no text anymore. Is this some incompability in my side or in purpose?  
2\. are you thinking about Integration scrobbling support?  
Sorry that I, as far as I think am not able to contribute in a productive way because I am no programmer, but never the less, my appreciations.  
Best!

**Comment by AdrienPoupa** [comment]
_From **HardCode4All** on **2019-03-01 07:25:19** (Migrated from Wordpress):_

Hi Adrien, do you know when the V0.20.2 will be available at F-Droid.org?

Best regards.

**Comment by AdrienPoupa** [comment]
_From **Micael** on **2019-04-03 17:01:01** (Migrated from Wordpress):_

Hi there, congrats on the efforts. Much appreciated. I suggest to include ability to add folders from a local server.

Cheers!

**Comment by AdrienPoupa** [comment]
_From **alex** on **2019-04-06 12:11:13** (Migrated from Wordpress):_

Hi,

I can’t find a way to update the music library successfully without going into a software reboot of my device. Am I missing something ?  
thx

**Comment by AdrienPoupa** [comment]
_From **Nae T'aan** on **2019-07-02 04:02:22** (Migrated from Wordpress):_

And plans for ChromeCast support?

**Comment by AdrienPoupa** [comment]
_From **Chuck** on **2019-08-03 00:24:44** (Migrated from Wordpress):_

Just dropping in to say that I love the Vinyl Music Player and use it as my only personal music player on Android (aside from NewPipe sometimes). Great work and thank you!

**Comment by AdrienPoupa** [comment]
_From **Scott Parsloe** on **2019-08-19 19:54:20** (Migrated from Wordpress):_

Hi Adrien,

This is my favourite music player app. The only thing it’s missing is the ability to cast to an audio speaker (like a Google Home). If it had that feature, I don’t think I’d ever use any other music player!

Keep up the good work! 🙂

**Comment by AdrienPoupa** [comment]
_From **Vrihub** on **2019-09-13 22:24:55** (Migrated from Wordpress):_

I’ve been using vinyl since a couple of weeks and I must confess it’s the best player I ever seen! A big THANK YOU!  
I hope to see it improve more and more (hence the flood of tickets that I just filed on github 🙂

**Comment by AdrienPoupa** [comment]
_From **Unknown** on **2019-10-19 11:12:09** (Migrated from Wordpress):_

I dont think the app works for Android 10.

**Comment by AdrienPoupa** [comment]
_From **chenn** on **2019-10-19 23:02:46** (Migrated from Wordpress):_

Love IT!

**Comment by AdrienPoupa** [comment]
_From **rayman95** on **2020-02-20 04:28:53** (Migrated from Wordpress):_

Hi, I love your app, but it will be perfect if you add support for cue+ape file.. actually it can only read the entire track….aimp and some paid are compatible…  
Thanks

**Comment by AdrienPoupa** [comment]
_From **Chuck** on **2020-05-18 17:27:27** (Migrated from Wordpress):_

I love your app! I use Vinyl pretty much exclusively to play my mp3, ogg, and flac files on my phone. If you’re accepting donated, I’d be happy to send you a few dollars or cryptocurrency.

**Comment by AdrienPoupa** [comment]
_From **Sergei** on **2021-09-30 21:59:56** (Migrated from Wordpress):_

Hello.  
Installed Vinyl and it almost satisfies my needs: simple shuffling player.  
But I found that it does not see the music on SD-card. For example, I browse to sdcard with file manager, then tap some file. OS offers me choice with several players I have.  
VLC: plays  
Google files: plays  
Built-in music player: plays  
Vinyl: completely ignorant

**Comment by AdrienPoupa** [comment]
_From **Nick Kontogouris** on **2022-04-12 00:26:34** (Migrated from Wordpress):_

Does feature a sleep timer and importing m3u8 lists? Thanx

**Comment by orava0** [comment]
Thank you for a very nice and simplistic player! In the settings, there is option for whitelisting. The setting is for /storage/emulated/0/Music and it can only be turned on/off. I`d like to change that folder to my microSD card where my music is located but I can`t find any way to change the whitelisting folder. Is there a way to do this?

Also, when I open the app, it takes me to the root of my microSD card in the folder view. Is there a setting to open the Music folder of my microSD card by default? This would save me from opening the correct folder everytime I use the app :-)
